include/point.h

```diff
@@ -9,19 +9,30 @@ template <unsigned Tdim>
 class Point {
  public:
   //! \brief Constructor with id and coordinates
-  Point(unsigned id, const std::array<double, Tdim>& coord);
+  Point(unsigned id, const std::array<double, Tdim>& coord,
```

Comment:
Not all points have stresses, like a vertex. You can derive a material point class.